### PR TITLE
make sure that the image to be used can be set from jenkins

### PR DESCRIPTION
**Pull request by metas-ts**
..also, use the latest master DB plus a bit of cleanup

### CODE REVIEW

File: docker-builds/Dockerfile.db-preloaded [hunk 1]
```diff
@@ -5,6 +5,8 @@ FROM metasfresh/metas-db:$REFNAME as db
 # https://cadu.dev/creating-a-docker-image-with-database-preloaded/
 # dirty but works
 
+# TODO: see how RunMigrationScriptsTest.java does it using our own java-tool. It's better for different reasons.
```
Comment (teosarca) [inline docker-builds/Dockerfile.db-preloaded:8]: @metas-ts (loud thinking)

u can even consider having a dedicated repo for `de.metas.migration` build it and deploy it as a docker image.
Then on this docker build we can grab the built jar from there.

File: .github/workflows/cicd.yaml [hunk 2]
```diff
@@ -355,6 +355,8 @@ jobs:
         with:
           domain: metasfresh
       - name: run-junit-tests-j8
+        env:
+          SKIP_MIGRATION_SCRIPTS_TEST: true # don't have RunMigrationScriptsTest.java apply the SQLs as that's already done further up
```
[the remark this answers is not in the document]
Comment (metas-ts) [reply .github/workflows/cicd.yaml:359]: @teosarca 
> somewhere here?

right; in run-junit-tests-j8

[the remark this answers is not in the document]
Comment (teosarca) [reply .github/workflows/cicd.yaml:359]: @metas-ts  but if we disable them, they won;t be tested there... so i expect to be tested somewhere else.
That "somewhere else" is the place i am looking for

**Comment by teosarca** [comment]
closing, legacy

**Pull request closed without merging**